### ISSUE TITLE
🐛 BUG: create cloudflare does not seed existing directory name

### ISSUE THREAD

**Issue by elithrar** [cloudflare/workers-sdk#3523]
### Which Cloudflare product(s) does this pertain to?

C3

### What version of `Wrangler` are you using?

3.1.1

### What operating system are you using?

macOS

### Describe the Bug

**Expectation**: invoking C3 with a directory name should create a new project with that name (or via `wrangler init --from-dash=NAME`)

**Actual**: it "shows" the name in the output, but if you press Enter, it throws and exits. A user needs to type the pre-populated name again.

```sh
‚ûú  npm create cloudflare@latest pg-demo --type="pre-existing" --existing-script="pg-demo"

using create-cloudflare version 2.0.10

‚ï≠ Create an application with Cloudflare Step 1 of 3
‚îÇ
‚ï∞ Where do you want to create your application? also used as application name
--pg-demo
Directory `` already exists and is not empty. Please choose a new name.
```

```sh
‚ûú    wrangler init --from-dash=pg-demo
 ‚õÖÔ∏è wrangler 3.1.1
------------------
Using npm as package manager.
‚úî Select an account ‚Ä∫ a single plum floating in perfume
‚ñ≤ [WARNING] The `init --from-dash` command is no longer supported. Please use `npm create cloudflare@2 pg-demo -- --type pre-existing --existing-script pg-demo` instead.

  The `init` command will be removed in a future version.

Running `npm create cloudflare@2 pg-demo -- --type pre-existing --existing-script pg-demo`...

using create-cloudflare version 2.0.10

‚ï≠ Create an application with Cloudflare Step 1 of 3
‚îÇ
‚ï∞ Where do you want to create your application? also used as application name
--pg-demo
Directory `` already exists and is not empty. Please choose a new name.
npm ERR! code 1
npm ERR! path /Users/matt/repos
npm ERR! command failed
npm ERR! command sh -c create-cloudflare "pg-demo" "--type" "pre-existing" "--existing-script" "pg-demo"

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/matt/.npm/_logs/2023-06-26T18_53_08_262Z-debug-0.log

‚úò [ERROR] Command failed with exit code 1: npm create cloudflare@2 pg-demo -- --type pre-existing --existing-script pg-demo
```

<img width="920" alt="image" src="https://github.com/cloudflare/workers-sdk/assets/18544/6215fa03-babe-42a7-a1fd-df3be6d4a23b">

**Comment by HyShai** [comment]
Not only that - the actual code from the existing Worker is never downloaded into the directory.

**Comment by TarasMakarchuk** [comment]
I tried to execute this command on an empty project, it creates without problems. If I try to create a worker in an existing project, I get the error "Directory ` ` already exists and is not empty. Please choose a new name".

**Comment by jculvey** [comment]
This was fixed by https://github.com/cloudflare/workers-sdk/pull/3525 and will be included in `2.0.12`.
